lib/nodes/addon/components/driver-oci/component.js

```diff
@@ -0,0 +1,212 @@
+  'BM.DenseIO2.52':          'BM.DenseIO2.52',
+  'BM.HPC2.36':              'BM.HPC2.36',
+  'VM.Standard.E2.1.Micro':  'VM.Standard.E2.1.Micro',
+  'VM.Standard.E2.2':        'VM.Standard.E2.2',
+  'VM.GPU2.1':               'VM.GPU2.1',
+  'VM.GPU2.2':               'VM.GPU2.2',
+  'VM.GPU3.1':               'VM.GPU3.1',
+  'VM.GPU3.2':               'VM.GPU3.2',
+  'VM.GPU3.4':               'VM.GPU3.4',
+  'VM.GPU3.8':               'VM.GPU3.8',
+}
+
+const imageMap = { 'Oracle-Linux-7.7': 'Oracle-Linux-7.7', }
+
+export default Component.extend(NodeDriver, {
+  intl:     service(),
+  settings: service(),
+
+  layout,
+
+  driverName:     'oci',
+  region:         'us-phoenix-1',
+  nodeImage:      'Oracle-Linux-7.7',
+  step:            1,
+
+  config: alias('model.ociConfig'),
+
+  init() {
+    this._super(...arguments);
+  },
+
+  actions: {
+    finishAndSelectCloudCredential(credential) {
+      set(this, 'model.cloudCredentialId', get(credential, 'id'))
+    }
+  },
+  regionChoices: Object.entries(regionMap).map((e) => ({
+    label: e[0],
+    value: e[1]
+  })),
+  selectedRegion: computed('config.region', function() {
+    const region = get(this, 'config.region');
+
+    return region;
+  }),
+  adChoices: computed('config.region', function() {
+    // TODO get these values dynamically from OCI API (/20160918/availabilityDomains)
+
+    const region = get(this, 'config.region')
+    var values
+
+    // 3 availability domains available
+    if (region === 'uk-london-1' || region === 'us-ashburn-1' || region === 'us-phoenix-1' || region === 'eu-frankfurt-1') {
+      values = {
+        'AD1': 'AD-1',
+        'AD2': 'AD-2',
+        'AD3': 'AD-3',
+      };
+    } else {
+      // 1 availability domain available
+      values = { 'AD1': 'AD-1', };
+    }
+
+    // get the keys
+    let keys = Object.keys(values);
+    // map the values into want you want
+    let result = keys.map((key) => {
+      return {
+        label: values[key],
+        value: values[key]
+      }
+    })
+
+    return result
+  }),
+  selectedAd: computed('config.nodeAvailabilityDomain', function() {
+    const ad = get(this, 'config.nodeAvailabilityDomain');
+
+    return ad;
+  }),
+  nodeShapeChoices: Object.entries(nodeShapeMap).map((e) => ({
+    label: e[1],
+    value: e[0]
+  })),
+  selectednodeShape: computed('config.nodeShape', function() {
+    const nodeShape = get(this, 'config.nodeShape');
+
+    return nodeShape && nodeShapeMap[nodeShape];
+  }),
+  imageChoices: Object.entries(imageMap).map((e) => ({
+    label: e[1],
+    value: e[0]
+  })),
+  selectedImage: computed('config.nodeImage', function() {
+    const nodeImage = get(this, 'config.nodeImage');
+
+    return nodeImage && imageMap[nodeImage];
+  }),
+  bootstrap() {
+    const config = get(this, 'globalStore').createRecord({
+      type:      'ociConfig',
+      region:    'us-phoenix-1',
+      nodeImage: 'Oracle-Linux-7.7',
+    });
```

Comment:
region and nodeImage are declared at the component level, i'd use them or get ride of them if you're only using these values here.